app/views/user_imports/form.html.erb

```diff
@@ -1,34 +1,5 @@
 <%= elmo_form_for(@user_import, html: {multipart: true}) do |f| %>
   <p class="instructions"><%= formatted_instructions %></p>
```

Comment:
`formatted_instructions` is a bad global method name when it applies only to user imports. Can you please move the code back into the template directly as it is with option set imports? We can maybe abstract out the instructions stuff later but let's get rid of that helper method.

Then I notice the only other method in that helper is not being used so let's nix it altogether. One less helper in the world! Yay!